config/initializers/devise.rb

```diff
@@ -127,15 +127,13 @@
 
   # ==> Configuration for :rememberable
   # The time the user will be remembered without asking for credentials again.
-  config.remember_for = 26.weeks
-  config.extend_remember_period = true
-  config.timeout_in = 20.weeks
```

Comment:
We don't use timeoutable, so I removed it from here, it's also not in right area of the file